hpx/parallel/executors/timed_execution_fwd.hpp

```diff
@@ -0,0 +1,209 @@
+//  Copyright (c) 2017 Hartmut Kaiser
+//
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+// hpxinspect:nounnamed
+
+#if !defined(HPX_PARALLEL_EXECUTORS_TIMED_EXECUTION_FWD_JAN_07_2017_0720AM)
+#define HPX_PARALLEL_EXECUTORS_TIMED_EXECUTION_FWD_JAN_07_2017_0720AM
+
+#include <hpx/config.hpp>
+#include <hpx/parallel/executors/execution_fwd.hpp>
+#include <hpx/traits/executor_traits.hpp>
+
+namespace hpx { namespace parallel { namespace execution
+{
+    ///////////////////////////////////////////////////////////////////////////
+    namespace detail
+    {
+        /// \cond NOINTERNAL
+        struct post_at_tag {};
+        struct post_after_tag {};
+        struct sync_execute_at_tag {};
+        struct sync_execute_after_tag {};
+        struct async_execute_at_tag {};
+        struct async_execute_after_tag {};
+
+        // forward declare customization point implementations
+        template <> struct customization_point<post_at_tag>;
+        template <> struct customization_point<post_after_tag>;
+        template <> struct customization_point<sync_execute_at_tag>;
+        template <> struct customization_point<sync_execute_after_tag>;
+        template <> struct customization_point<async_execute_at_tag>;
+        template <> struct customization_point<async_execute_after_tag>;
+        /// \endcond
+    }
+
+    // define customization points
+    namespace
+    {
+        ///////////////////////////////////////////////////////////////////////
+        // extensions
+
+        // NonBlockingOneWayExecutor customization points: execution::post_at
+        // and execution::post_after
+
+        /// Customization point of asynchronous fire & forget execution agent
+        /// creation supporting timed execution.
+        ///
+        /// This asynchronously (fire & forget) creates a single function
+        /// invocation f() using the associated executor at the given point in
+        /// time.
+        ///
+        /// \param exec [in] The executor object to use for scheduling of the
+        ///             function \a f.
+        /// \param abs_time [in] The point in time the given function should be
+        ///             scheduled at to run.
+        /// \param f    [in] The function which will be scheduled using the
+        ///             given executor.
+        /// \param ts... [in] Additional arguments to use to invoke \a f.
+        ///
+        /// \note This calls exec.apply_execute_at(abs_time, f, ts...), if
```

Comment:
shouldn't this read `/// \note This calls exec.post_at(abs_time, f, ts...), if`?